NuGet.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This should rather managed on a solution level.

Reply:
Sorry I didn't mean to commit this. I'll remove it.